src/settings/environment.inc

```diff
@@ -90,20 +90,20 @@ elseif (isset($_ENV['PLATFORM_APPLICATION'])) {
   }
 }
 elseif (getenv('DOCKER_LOCAL')) {
-    // Local environment settings - see 90-docker-settings.inc.
-    $platform = 'docker';
-    $env = $base_domains[$_SERVER['HTTP_HOST']];
-    $instance = $_SERVER['HTTP_HOST'];
+  // Local environment settings - see 90-docker-settings.inc.
+  $platform = 'docker';
```

Comment:
Shouldn't this be D_PLATFORM_DOCKER?

Reply:
Yes, it should be.
I've since spotted this issue and I have fixed in the testing + Drush 9 PR  here https://github.com/teamdeeson/d8-quickstart/pull/68

Reply:
Do you mean PR #69 ?

Reply:
Darn, yeah, I do